extensions/amp-mustache/0.2/amp-mustache.js

```diff
@@ -137,12 +137,12 @@ export class AmpMustache extends BaseTemplate {
    * @private
    */
   purifyAndSetHtml_(html) {
-    const body = this.purifier_.purifyHtml(html);
-    // TODO(choumx): Remove innerHTML usage once DOMPurify bug is fixed.
-    // https://github.com/cure53/DOMPurify/pull/295
-    const root = this.win.document.createElement('div');
-    root./*OK*/ innerHTML = body./*OK*/ innerHTML;
-    return this.unwrap(root);
+    const body = this.purifier_.purifyHtml(
+      `<div>${html}</div>`,
+      this.win.document
```

Comment:
You can remove this, the document is specified in the constructor, so `purifyHtml` only takes one argument now.